code/game/objects/structures/monster_spawner.dm

```diff
@@ -74,3 +75,86 @@
 	name = "basilisk den"
 	desc = "A den housing a nest of basilisks, bring a coat."
 	mob_types = list(/mob/living/simple_animal/hostile/asteroid/basilisk)
+
+/obj/structure/spawner/sentient
+	var/role_name = "A sentient mob"
+	var/assumed_control_message = "You are a sentient mob from a badly coded spawner"
+
+/obj/structure/spawner/sentient/Initialize(mapload)
+	. = ..()
+	notify_ghosts("A [name] has been created in \the [get_area(src)]!", source = src, title = "Sentient Spawner Created", flashwindow = FALSE
+	)
+
+/obj/structure/spawner/sentient/on_mob_spawn(atom/created_atom)
+	created_atom.AddComponent(\
+		/datum/component/ghost_direct_control,\
+		role_name = src.role_name,\
+		assumed_control_message = src.assumed_control_message,\
+		after_assumed_control = CALLBACK(src, PROC_REF(became_player_controlled)),\
+	)
+
+/obj/structure/spawner/sentient/proc/became_player_controlled(mob/proteon)
+	return
+
+/obj/structure/spawner/sentient/proteon_spawner
+	name = "eldritch gateway"
+	desc = "A dizzying structure that somehow links into Nar'Sie's own domain. The screams of the damned echo continously."
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "hole"
+	light_power = 2
+	light_color = LIGHT_COLOR_RED
+	max_integrity = 50
+	density = FALSE
+	max_mobs = 2
+	spawn_time = 15 SECONDS
+	mob_types = list(/mob/living/simple_animal/hostile/construct/proteon/hostile)
+	spawn_text = "arises from"
+	faction = list("cult")
+	role_name = "A proteon cult construct"
+	assumed_control_message = null
+	var/gps
+
+/obj/structure/spawner/sentient/proteon_spawner/Initialize(mapload)
+	. = ..()
+	GLOB.proteon_portals += src
+	gps = new /obj/item/gps/internal/proteon(src)
+
+/obj/structure/spawner/sentient/proteon_spawner/Destroy()
+	GLOB.proteon_portals -= src
+	QDEL_NULL(gps)
+	return ..()
+
+/obj/structure/spawner/sentient/proteon_spawner/examine_status(mob/user)
+	if(IS_CULTIST(user) || !isliving(user))
+		return "<span class='cult'>It's at <b>[round(obj_integrity * 100 / max_integrity)]%</b> stability.</span>"
+	return ..()
+
+/obj/structure/spawner/sentient/proteon_spawner/examine(mob/user)
+	. = ..()
+	if(!IS_CULTIST(user) && isliving(user))
+		var/mob/living/living_user = user
+		living_user.adjustBrainLoss(5)
+		. += "<span class='danger'>The voices of the damned echo relentlessly in your mind, continously rebounding on the walls of your self the more you focus on [src]. Your head pounds, better keep away...</span>"
```

Comment:
Recommending userdanger since it deals brain damage.
You might also want to add a cooldown/status check on the brain damage infliction so people don't just shift click it a bunch and fall over dead
```suggestion
		. += "<span class='userdanger'>The voices of the damned echo relentlessly in your mind, continously rebounding on the walls of your self the more you focus on [src]. Your head pounds, better keep away...</span>"
```

Reply:
Will commit it, if someone examines it 20 times it's _probably_ on them but that is my thought